### PR TITLE
Put logs in CI

### PR DESCRIPTION
**Pull request by TingDaoK**
*Issue #, if available:*

*Description of changes:*
- Always get error level log enabled, and print it out after CI
- Lower the number of requests we made for concurrent request test, since it's throttled by server, and failed occasionally. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: s3-native-client/src/test/java/com/amazonaws/s3/S3NativeClientTest.java [hunk 1]
```diff
@@ -55,6 +56,7 @@
     @BeforeClass
     public static void haveAwsCredentials() {
         Assume.assumeTrue(areAwsCredentialsAvailable());
+        Log.initLoggingToFile(Log.LogLevel.Error, "log.txt");
```
Comment (bretambrose) [inline s3-native-client/src/test/java/com/amazonaws/s3/S3NativeClientTest.java:59]: Couldn't we do this in a before-traited function in AwsClientTestFixture?

File: builder.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
         "mvn -P continuous-integration -B compile"
     ],
     "test_steps": [
-        "mvn -B test -DredirectTestOutputToFile=true -DforkCount=0 -DrerunFailingTestsCount=5"
+       "aws-crt-java-test"
```
Comment (bretambrose) [inline builder.json:10]: What is this?

Comment (TingDaoK) [reply builder.json:10]: that's using the python script to run test. https://github.com/awslabs/aws-crt-java/pull/346/files#diff-ce63a05ed0ed2cf8fc220bbf3cf6fc06e4034fae5fe8be8404060bfaf1bf0d98
So, in order to print the log out, we need to continue the command even if the first cmd failed. Get around it by two different `os.system()`.

Comment (bretambrose) [reply builder.json:10]: Is there a builder version bump needed?  The script is using underscores, this is using dashes.

Comment (TingDaoK) [reply builder.json:10]: Nope, the builder will find the script based on the name https://github.com/awslabs/aws-crt-java/pull/346/files#diff-ce63a05ed0ed2cf8fc220bbf3cf6fc06e4034fae5fe8be8404060bfaf1bf0d98R18

**Review by bretambrose — APPROVED** [review]

**Pull request merged**